.goreleaser.yaml

```
@@ -1,7 +1,8 @@
 version: 2
+project_name: test-engine-client
 
 release:
-  name_template: Test Splitter v{{.Version}}
+  name_template: Buildkite Test Engine Client v{{.Version}}
```

```suggestion
  name_template: Test Engine Client v{{.Version}}
```
I don't think this is necessary. This is the title for GitHub release. The repo is already under Buildkite namespace.